modules/monitoring-configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-different-alert-receivers-for-default-platform-alerts-and-user-defined-alerts_{context}"]
+= Configuring different alert receivers for default platform alerts and user-defined alerts
+
+You can configure different alert receivers for default platform alerts and user-defined alerts to ensure the following results:
```

Comment:
We should note that it doesn't apply if you've configured a dedicated Alertmanager for user-defined alerts (https://docs.openshift.com/container-platform/4.15/monitoring/enabling-alert-routing-for-user-defined-projects.html#understanding-alert-routing-for-user-defined-projects_enabling-alert-routing-for-user-defined-projects).